app/models/feedback.rb

```diff
@@ -0,0 +1,4 @@
+class Feedback < ActiveRecord::Base
```

Comment:
To get the "annotations that all other models have, just run `rake annotate` in your console (it's a gem we use).